src/libraries/Native/build-native.cmd

```diff
@@ -127,6 +127,11 @@ may help to copy its "DIA SDK" folder into "%VSINSTALLDIR%" manually, then try a
 exit /b 1
 
 :GenVSSolution
+:: generate version file
+powershell -NoProfile -ExecutionPolicy ByPass -NoLogo -File "%__repoRoot%\eng\common\msbuild.ps1" /clp:nosummary %__ArcadeScriptArgs%^
```

Comment:
I wonder why was the move from the build-native.proj to this place needed. I am no msbuild expert, so I am just curious.

Reply:
Also, I cannot see similar change in the build-native.sh, so it looks suspicious.

Reply:
For Unix, this code is moved to `eng/native/build-commons.sh` in `build_native()` function. I haven't done much work on Windows scripts in this PR, this was just to align it with `coreclr/build.cmd`. In future we can consolidate batch scripts as well.